keras_nlp/src/models/efficientnet/mbconv.py

```diff
@@ -0,0 +1,235 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import keras
+
+BN_AXIS = 3
+
+CONV_KERNEL_INITIALIZER = {
+    "class_name": "VarianceScaling",
+    "config": {
+        "scale": 2.0,
+        "mode": "fan_out",
+        "distribution": "truncated_normal",
+    },
+}
+
+
+class MBConvBlock(keras.layers.Layer):
+    def __init__(
+        self,
+        input_filters,
+        output_filters,
+        expand_ratio=1,
+        kernel_size=3,
+        strides=1,
+        se_ratio=0.0,
+        bn_momentum=0.9,
+        activation="swish",
+        survival_probability: float = 0.8,
+        **kwargs
+    ):
+        """
+        Implementation of the MBConv block (Mobile Inverted Residual Bottleneck)
```

Comment:
same, one liner on the quote line, and keep it short

Reply:
done